routes/products.js

```diff
@@ -1,24 +1,32 @@
 const productsRouter = require("express").Router();
 const { getProductById, getAllProducts, createProduct, getProductByName } = require("../db")
-
+const { 
+    getProductById, 
+    getAllProducts, 
+    createProduct, 
+    getProductByName } = require("../db")
+//get all products   
 productsRouter.get("/", async (req, res, next) =>{
+    console.log("Got here!")
```

Comment:
if you are confident this is working, could get rid of this log

Reply:
complete